website/src/categories/getting-started/developers.md

```diff
@@ -60,36 +77,34 @@ To get started using the SL Design System, you can install it via npm. Each comp
 
 ## Setup a theme
 
-To start using the SL Design System, you need to setup a theme for your application. The theme determines how your application looks. There is a theme for every major product within Sanoma Learning. You can install a theme as by installing the NPM package, in this example we use the `sanoma-learning` theme.
+To start using the SL Design System, you need to setup a theme for your application. The theme determines how your application looks. There is a theme for every major product within Sanoma Learning.
 
-<div class="ds-code">
+Please note that the webfonts of your application are not a part of the theme. There are different ways of loading the fonts an application used across the products in Sanoma Learning. If is up to you to load the webfonts in your application.
 
-  ```bash
-  npm install @sl-design-system/sanoma-learning
-  ```
-
-</div>
-
-After installing the theme, you need to import the theme in your application. If you are working in an Angular application you can do this by loading the theme in your application in your angular.json, in all the places where you include your other css files:
+You can install a theme as by installing the NPM package, in this example we use the `sanoma-learning` theme.
 
 <div class="ds-code">
 
-  ```json
-   './node_modules/@sl-design-system/sanoma-learning/all.css'
+  ```bash
+  npm add @sl-design-system/sanoma-learning
```

Comment:
in other places we also offer the yarn version.

(this part can eventually be replaced by the new component i made)